types/rest/rest.go

```diff
@@ -197,6 +197,22 @@ func ParseFloat64OrReturnBadRequest(w http.ResponseWriter, s string, defaultIfEm
 	return n, true
 }
 
+// ParseQueryHeightOrReturnBadRequest sets the height to execute a query if set by the http request.
+func ParseQueryHeightOrReturnBadRequest(w http.ResponseWriter, cliCtx context.CLIContext, r *http.Request) (context.CLIContext, bool) {
```

Comment:
Can we add a note to the godoc on what the boolean signifies?

Reply:
We can also add a name to the boolean